pkg/minikube/registry/drvs/docker/docker.go

```diff
@@ -136,6 +136,9 @@ func status() (retState registry.State) {
 	}()
 
 	versions := strings.Split(string(o), ":")
+	if len(versions) < 2 {
```

Comment:
maybe we can add a unit test case in TestCheckDockerEngineVersion ?
https://github.com/medyagh/minikube/blob/99756bef0b9a90b9097668cb4e9c255f387cbadd/pkg/minikube/registry/drvs/docker/docker_test.go#L67

Reply:
Added units tests